packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -911,6 +915,16 @@ class AnimatedAlign extends ImplicitlyAnimatedWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// If non-null, sets its height to the child's height multiplied by this factor.
+  ///
+  /// Must be within the range 0.0 - 1.0. Defaults to 1.0.
```

Comment:
I thought these could be greater than 1.0?

Reply:
They can 😬  Just pushed the correction :)